nav-app/src/app/datatable/data-table.component.html

```diff
@@ -41,32 +41,13 @@
             <div *ngIf="configService.getFeature('search')" class="control-row-item">
 
                 <div class="control-row-button dropdown noselect"
```

Comment:
The dropdown class should be removed here (which _should_ remove the arrow after the button) since it's not really a dropdown anymore.